app/scripts/lib/validate.js

```diff
@@ -16,6 +16,8 @@ define(function (require, exports, module) {
   // taken from the fxa-oauth-server
   const B64URL_STRING = /^[A-Za-z0-9-_]+$/;
 
+  const B32_STRING = /^[0123456789ABCDEFGHJKMNPQRSTVWXYZ]+$/;
```

Comment:
No biggie, but I notice this could be expressed more succinctly, even accounting for the skipped characters:

```
/^0[-9A-HJKM-Z]+$/
```